MAPL_engine/mongo_plugin.go

```diff
@@ -243,3 +243,31 @@ func removeQuotes(str string) (string, error) {
 
 	return str, nil
 }
+
+//-------------------------------------------------------------------------------------
+
+const AggregateQuery = "aggregate"
+const SimpleQuery = "query"
```

Comment:
Please separate this into a another file
&
Please create new type:
type QueryType string

const (
QueryTypeSimple QueryType  = "simple"
QueryTypeAggregate QueryType  = "aggregate"
)

type MaplToMongoResult struct {
	Type QueryType
	Query interface{}
}